### PR TITLE
Update website with maf

### PR DESCRIPTION
**Pull request by mpage21**
Made some other changes as well

### CODE REVIEW

File: static/js/calculate_score.js [hunk 1]
```diff
@@ -461,7 +445,7 @@ var calculatePolyScore = async () => {
 
         //if text input is empty, return error
         if (!textArea.value) {
-            updateResultBoxAndStoredValue("Please input RS IDs by hand according to the procedures above or import a VCF file using the \"File Upload\" and then the \"Choose File\" buttons above (step 1).");
```
Comment (MattCloward) [inline static/js/calculate_score.js:464]: Can we do both? Just in case the user has pop ups disabled?

File: static/js/calculate_score.js [hunk 2]
```diff
@@ -913,83 +906,79 @@ var calculateScore = async (associationData, mafData, clumpsData, greppedSamples
...
-                                                        resultObj[printStudyID][trait][pValBetaAnnoValType][individualName]['snps'][key]['MAF'] = mafData[key]['alleles'][associationObj.riskAllele]
-                                                    }
-                                                }
                                             }
                                         }
-                                    }
-                                    else {
-                                        //TODO: add something to identify snps that aren't added because of pValue
+                                        else {
+                                            //TODO: add something to identify snps that aren't added because of pValue
```
Comment (MattCloward) [inline static/js/calculate_score.js:980]: Can this TODO be resolved?

File: static/js/calculate_score.js [hunk 3]
```diff
@@ -1062,10 +1047,10 @@ var calculateScore = async (associationData, mafData, clumpsData, greppedSamples
         //if the input data doesn't have an individual in it (we can assume this is a text input query with no matching SNPs)
         //TODO fill this out
```
Comment (MattCloward) [inline static/js/calculate_score.js:1048]: Can this TODO be removed?

**Review by MattCloward — CHANGES_REQUESTED** [review]
See my few comments

File: static/downloadables/parse_associations.py [hunk 4]
```diff
@@ -475,6 +475,9 @@ def parse_vcf(filteredFilePath, clumpsObjDict, tableObjDict, possibleAlleles, sn
                                                         index_snp_map[sample][clumpNum] = rsID, riskAllele, alleles if complements is None else complements
                                                         clumpedVariants.add(index_snp)
                                                     else:
+							# TODO: What if the current SNP also has two missing alleles? In this case, wouldn't we
```
Comment (lizvance) [inline static/downloadables/parse_associations.py:478]: @mpage21  @MattCloward Can you guys take a look at this TODO comment? I think I may have found something that needs to change in the clumping logic, but want to make sure I'm not going crazy.

Comment (mpage21) [reply static/downloadables/parse_associations.py:502]: I think that is a valid change to be made.

File: static/js/calculate_score.js [hunk 5]
```diff
@@ -907,89 +1015,110 @@ var calculateScore = async (associationData, mafData, clumpsData, greppedSamples
...
+                                                                resultObj[printStudyID][trait][pValBetaAnnoValType][individualName]['snps'][key]['numAllelesMatch'] = numAllelesMatch
+                                                                resultObj[printStudyID][trait][pValBetaAnnoValType][individualName]['snps'][key]['numAlleleMissingGenotype'] = numAlleleMissingGenotype
+                                                                if (numAlleleMissingGenotype > 0) {
+                                                                    if (key in mafData && riskAllele in mafData[key]['alleles']){
+                                                                        resultObj[printStudyID][trait][pValBetaAnnoValType][individualName]['snps'][key]['MAF'] = mafData[key]['alleles'][riskAllele]
+                                                                    }
+                                                                }
+                                                                indexSnpObj[traitStudypValueAnnoValTypeSamp][clumpNum] = [key, riskAllele]
+                                                            } else {
```
[the remark this answers is not in the document]
Comment (mpage21) [reply static/js/calculate_score.js:1088]: Is the stuff below supposed to be in it?

File: static/calculate_score.html [hunk 6]
```diff
@@ -293,20 +282,17 @@ <h5>1: Enter P-Value Cutoff&nbsp;</h5>
                 </div>
               </div>
 
+              <!--TODO update with CLUMPING-->
```
Comment (MattCloward) [inline static/calculate_score.html:285]: Has this been done in a different branch?

**Review by MattCloward — APPROVED** [review]
This looks great!

File: static/js/calculate_score.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (lizvance) [reply static/js/calculate_score.js:1088]: Oh whoops. Yes. Thanks for catching that!

**Pull request merged**